src/html_support_files/odoc.css

```diff
@@ -1228,55 +1249,33 @@ td.def-doc *:first-child {
   body.odoc {
     max-width: 132ex;
     grid-template-areas:
-    "search-bar nav"
-    "sidebar    preamble"
-    "sidebar    content";
-  }
-
-  body.odoc:has(> .odoc-search:focus-within) {
-    grid-template-areas:
-    "nav        nav"
-    "search-bar search-bar"
-    "sidebar    preamble"
-    "sidebar    content";
-  }
-
-  nav.odoc-nav:has(+ .odoc-search:focus-within) {
-    padding-bottom: 0;
-  }
-
-  .odoc-tocs {
+      "search-bar nav"
+      "sidebar    preamble"
+      "sidebar    content";
+    .odoc-tocs {
       display: flex;
       grid-area: sidebar;
       flex-direction : column;
       gap: 20px;
-  }
-  .odoc-toc {
-      position: unset;
-      max-height: unset;
+      .odoc-toc {
+        position: unset;
+        max-height: unset;
+      }
```

Comment:
What is this rule doing in another one? Am I too old and this is valid?

Reply:
Yes, it is valid, and quite cool: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting
Thanks for teaching me that!

However, [caniuse](https://caniuse.com/?search=css%20nesting) reports only 82% of used browser with full support. Since this was not used before, is it really worth introducing it?

Reply:
I was used before, the whole media rule was nested.

Reply:
Media queries are not CSS nesting... They have 98% support on [caniuse](https://caniuse.com/css-mediaqueries).